packages/grid/_modules_/grid/components/cell/GridRowCells.tsx

```diff
@@ -90,7 +90,9 @@ export const GridRowCells = React.memo(function GridRowCells(props: RowCellsProp
     }
 
     if (getCellClassName) {
-      classNames.push(getCellClassName(cellParams));
+      const classNameParams = { ...cellParams };
```

Comment:
```suggestion
      const classNameParams = { api, ...cellParams };
```

This makes the `delete classNameParams.api` unnecessary.

Reply:
https://github.com/mui-org/material-ui-x/pull/2312/files#diff-b58ef2608987589ff71f621df6cc9458c607e9b080c9a2c9be7dcaf920539f67R111

In that case we may want to use delete here